.github/workflows/windows-latest-mingw.yml

```diff
@@ -50,8 +50,10 @@ jobs:
       # Configure CMake in a 'build' subdirectory
       run: |
         cmake \
+          -G "MinGW Makefiles"
```

Comment:
MinGW Makefiles are sometimes a bit strange. "MSYS Makefiles" might be better.
But preferably you'd use the default generator for reset build environment, i.e., `-GNinja`.